recipes-rte/rte-tests/files/cukinia.conf

```diff
@@ -0,0 +1,47 @@
+export CUKINIA_ROOT=/etc/cukinia
+
+cukinia_kconf GPIO_SYSFS y
+cukinia_kconf SPI_SPIDEV y
+cukinia_kconf PINCTRL_MCP23S08 m
+
+# GPIO expander
+cukinia_test -d /sys/class/gpio/gpiochip400
+
+cukinia_test -d /sys/class/gpio/gpio199
+cukinia_test -d /sys/class/gpio/gpio11
+cukinia_test -d /sys/class/gpio/gpio12
+cukinia_test -d /sys/class/gpio/gpio199
+cukinia_test -d /sys/class/gpio/gpio400
+cukinia_test -d /sys/class/gpio/gpio401
+cukinia_test -d /sys/class/gpio/gpio402
+cukinia_test -d /sys/class/gpio/gpio403
+cukinia_test -d /sys/class/gpio/gpio404
+cukinia_test -d /sys/class/gpio/gpio405
+cukinia_test -d /sys/class/gpio/gpio406
+cukinia_test -d /sys/class/gpio/gpio407
+cukinia_test -d /sys/class/gpio/gpio408
+cukinia_test -d /sys/class/gpio/gpio409
+cukinia_test -d /sys/class/gpio/gpio410
+cukinia_test -d /sys/class/gpio/gpio411
+cukinia_test -d /sys/class/gpio/gpio412
+cukinia_test -d /sys/class/gpio/gpio413
+cukinia_test -d /sys/class/gpio/gpio414
+cukinia_test -d /sys/class/gpio/gpio415
+cukinia_test -d /sys/class/gpio/gpio6
+
+cukinia_test -c /dev/spidev1.0
+
+cukinia_cmd ser2net -v
+cukinia_cmd flashrom --version
+cukinia_test -d /data
+# TODO: test for support for specific chips we care about (such as new ones for
```

Comment:
@PLangowski Please fill this in, possibly also with other tests if you come up with something more.

Reply:
@PLangowski This still needs to be done. We need to verify that flashrom supports all chips requested in the issue.